api-doc.yaml

```diff
@@ -484,34 +484,16 @@ paths:
               $ref: '#/components/responses/UnauthorizedErrorResponse'
         '404':
               $ref: '#/components/responses/NotFoundErrorResponse'
-  /members/{member-id}:
```

Comment:
There is no need for an endpoint to fetch a specific member since only the project member's role can be updated, or the member can be deleted. On the frontend, a list of all members will be displayed, and users can directly update the role or delete members from this list